Source/WebKit/Platform/IPC/StreamConnectionWorkQueue.cpp

```diff
@@ -79,14 +79,15 @@ void StreamConnectionWorkQueue::removeStreamConnection(StreamServerConnection& c
     wakeUp();
 }
 
-void StreamConnectionWorkQueue::stopAndWaitForCompletion()
+void StreamConnectionWorkQueue::stopAndWaitForCompletion(std::optional<WTF::Function<void()>>&& cleanup)
```

Comment:
A WTF::Function can already be null so we shouldn't wrap it in an optional.

Just use WTF::Function<void()>&&